bestofjs/bestofjs
# Add OG image for monthly rankings pages

## Goal

The monthly rankings page deserve their own OG image too, to be able to preview the rankings when sharing URLs `/rankings/monthly/2023/10`.

This PR adds the image and specific meta data.

- title: `Best of JS • Rankings October 2023`
- description: `The most popular projects on Best of JS in October 2023: Bruno, Wasp, shadcn/ui, Excalidraw, Bun, Next.js, Payload, create-t3-app, Supabase, htmx...`

## How to test

- Click to "Monthly" in the top menu to go to rankings page
- Alternative: from home top page, inside the "Rankings October 2023" block click on "View monthly rankings" link
- Check the page metadata when inspecting the page `<head>`

## Screenshots

![image](https://github.com/bestofjs/bestofjs/assets/5546996/35ef44fa-cbeb-4606-8091-70f2bec1bbd0)
